### PR TITLE
New sprites and changes 

### PR DESCRIPTION
**Pull request by CommissarBane**
Added the combiflamer and the flamer sprites in the game
Added a slight debuff to range and damage resistance for the MK5
Added the infernus pistol sprite in the game
Added the Meltagun into the game
Added the Knife sprite

### CODE REVIEW

File: objects/obj_controller/Step_0.gml [hunk 1]
```diff
@@ -466,13 +466,15 @@ if (menu=1) and (managing>0){
...
                 
-                if (ma_armour[sel]="MK3 Iron Armour") then ra-=0.1;
-                if (ma_armour[sel]="MK4 Maximus"){ra+=0.05;ma+=0.05;}
-                if (ma_armour[sel]="MK5 Heresy"){ma+=0.1;}
-                if (ma_armour[sel]="MK6 Corvus"){ra+=0.1;}
-                if (string_count("Artificer",ma_armour[sel])>0){ma+=0.1;}
-                if (string_count("Terminator",ma_armour[sel])>0){ra-=0.1;ma+=0.2;}
-                if (ma_armour[sel]="Tartaros"){ra-=0.05;ma+=0.2;}
+
+                if (ma_armor[sel]="MK3 Iron Armor") then ra-=0.1;
```
Comment (GalacticChimp) [inline objects/obj_controller/Step_0.gml:470]: please change these to armo**u**r, we just merged an armor -> armour rewrite

Comment (CommissarBane) [reply objects/obj_controller/Step_0.gml:470]: I must have missed that

**Comment by GalacticChimp** [comment]
45 instances of armor still left:

image

**Comment by GalacticChimp** [comment]
also, next time I highly recommend you create a dedicated branch for your feature/bugfix/change. If you commit your changes directly to main, you'll quickly run into issues with merging ZAB909:main into your repo

**Comment by GalacticChimp** [comment]
> > 45 instances of armor still left:
> > image
> 
> Interesting I can't seem to see them on a phone. Would fix them when I get home. Also sorry for putting you to look over and over again.

np, it's just a quick search in VSCode

**Comment by CommissarBane** [comment]
I think that's all(hope to not have missed one of them)

**Comment by GalacticChimp** [comment]
@CommissarBane just 1 tiny revert, please make Armourum Ferrum -> Armo~~u~~rum Ferrum, that's the one exception straight from Lexicanum: https://wh40k.lexicanum.com/wiki/Aquila#Great_Crusade.2FPre-Heresy (4th image in the linked section). After this 1 change I'll approve and merge immediately

**Comment by CommissarBane** [comment]
> @CommissarBane just 1 tiny revert, please make Armourum Ferrum -> Armo~u~rum Ferrum, that's the one exception straight from Lexicanum: https://wh40k.lexicanum.com/wiki/Aquila#Great_Crusade.2FPre-Heresy (4th image in the linked section). After this 1 change I'll approve and merge immediately

Sorry bout that. One second

**Review by GalacticChimp — APPROVED** [review]

**Pull request merged**